ui/src/app/pages/login/account-creation/account-creation-survey.tsx

```diff
@@ -143,37 +147,59 @@ export class AccountCreationSurvey extends React.Component<AccountCreationSurvey
       </Section>
 
       {/*Ethnicity section*/}
-      <DropDownSection header='2. Ethnicity' options={AccountCreationOptions.Ethnicity}
+      <DropDownSection header='2. Ethnicity' options={AccountCreationOptions.ethnicity}
                        value={demographicSurvey.ethnicity}
                        onChange={(e) => this.updateDemographicAttribute('ethnicity', e)}/>
 
       {/*Gender section*/}
       <Section header='3. Gender'>
```

Comment:
Do we want to update this to Gender Identity ?

Reply:
That's a text change. It sounds like there may be a lot of text changes on these pages, so may want to wait until those conversations are finished.